controllers/articles_controllers.js

```diff
@@ -33,4 +33,17 @@ async function getArticles(req, res, next) {
     }
 }
 
-module.exports = {getArticleById, getArticles, getArticleCommentsById}
\ No newline at end of file
+async function postArticleCommentById(req, res, next) {
+    const { article_id } = req.params
+    const newComment = req.body
+    try {
+        await fetchArticleById(article_id)
+        const comment = await insertArticleCommentById(article_id, newComment)
+        res.status(201).send({comment})
+    } 
+    catch (error) {
+        next(error)
+    }
+}
```

Comment:
Using async await is really cool and your use of it is working well. Generally, if your tests are using .then promises, then your code should also use it too. No need to change it now, but it might be something you implement following this week? Refactoring your tests to use async await / refactoring your code to use .then. It's completely up to you :)